### PR TITLE
No Lombok.

### PR DESCRIPTION
**Pull request by jonspalmer**
Mostly used the IntelliJ plugin to delombok.

### CODE REVIEW

File: hibernate-redis/build.gradle [hunk 1]
```diff
@@ -25,8 +25,6 @@ dependencies {
     compile group: 'org.xerial.snappy', name: 'snappy-java', version:'1.1.2.1'
     compile group: 'de.ruedigermoeller', name: 'fst', version:'2.45'
     compile group: 'org.slf4j', name: 'slf4j-api', version:'1.7.7'
-    compile group: 'ch.qos.logback', name: 'logback-classic', version:'1.1.2'
```
Comment (jonspalmer) [inline hibernate-redis/build.gradle:28]: killed logback dependencyt too.

File: hibernate-redis/src/main/java/org/hibernate/cache/redis/serializer/BinaryRedisSerializer.java [hunk 2]
```diff
@@ -30,18 +30,18 @@
...
     public byte[] serialize(final T graph) {
         if (graph == null) return EMPTY_BYTES;
 
-        try {
-            @Cleanup
+        try (
             ByteArrayOutputStream os = new ByteArrayOutputStream();
-            @Cleanup
             ObjectOutputStream oos = new ObjectOutputStream(os);
+        ) {
```
Comment (jonspalmer) [inline hibernate-redis/src/main/java/org/hibernate/cache/redis/serializer/BinaryRedisSerializer.java:44]: Java 7 has nice try-with support.

File: hibernate-redis/src/main/java/org/hibernate/cache/redis/regions/RedisDataRegion.java [hunk 3]
```diff
@@ -179,4 +190,8 @@ public long nextTimestamp() {
     public int getTimeout() {
         return cacheLockTimeout;
     }
+
+    public RedisAccessStrategyFactory getAccessStrategyFactory() {
```
Comment (johnou) [inline hibernate-redis/src/main/java/org/hibernate/cache/redis/regions/RedisDataRegion.java:194]: protected?

Comment (johnou) [inline hibernate-redis/src/main/java/org/hibernate/cache/redis/regions/RedisDataRegion.java:194]: let's revisit visibility in a follow up pull request before making next release.

**Pull request merged**